.github/workflows/pypi_release.yml

```diff
@@ -30,5 +30,31 @@ jobs:
         TWINE_PASSWORD: ${{ secrets. PYPI_PASSWORD }}
         RELEASE: 1
       run: |
+        cd core/
         python setup.py sdist bdist_wheel
         twine upload dist/* --verbose
+        cd ..
+        cd tabular/
+        python setup.py sdist bdist_wheel
+        twine upload dist/* --verbose
+        cd ..
+        cd mxnet/
+        python setup.py sdist bdist_wheel
+        twine upload dist/* --verbose
+        cd ..
+        cd extra/
+        python setup.py sdist bdist_wheel
+        twine upload dist/* --verbose
+        cd ..
+        cd text/
+        python setup.py sdist bdist_wheel
+        twine upload dist/* --verbose
+        cd ..
+        cd vision/
+        python setup.py sdist bdist_wheel
+        twine upload dist/* --verbose
+        cd ..
+        cd autogluon/
+        python setup.py sdist bdist_wheel
+        twine upload dist/* --verbose
```

Comment:
Can you briefly spell out what'll happen when user executes `pip install autogluon`? (once everything's on pypi)

Reply:
It's going to install all the packages in [requirements](https://github.com/gradientsky/autogluon/blob/modules-v2/autogluon/setup.py#L43-L73) including all the sub-modules.